x-pack/test/functional/apps/ml/data_frame_analytics/index.ts

```diff
@@ -12,9 +12,9 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     this.tags(['mlqa', 'skipFirefox']);
 
     loadTestFile(require.resolve('./outlier_detection_creation'));
-    loadTestFile(require.resolve('./regression_creation'));
-    loadTestFile(require.resolve('./classification_creation'));
-    loadTestFile(require.resolve('./cloning'));
-    loadTestFile(require.resolve('./feature_importance'));
+    // loadTestFile(require.resolve('./regression_creation'));
```

Comment:
Are these edits meant to be included?

Reply:
Thanks for catching that, committed accidentally, removed in 7ad9b578bbe1ac050ad1c583238b92e6eea58df4